arches/app/templates/views/components/search/search-results.htm

```diff
@@ -21,63 +25,73 @@
         <div class="search-listing-footer">
             <div style="flex-grow: 1;">
                 <!--ko if: provisional_resource == 'true' -->
-                <a class="provisional-edits" href="#">
+                <span class="provisional-edits">
                     <i class="fa fa-exclamation-circle"></i>
                     <span data-bind="text: $root.translations.provisional"></span>
-                </a>
+                </span>
                 <!--/ko-->
 
                 <!--ko if: provisional_resource == 'partial' && $parent.userIsReviewer -->
-                <a class="provisional-edits" href="#">
+                <span class="provisional-edits">
                     <i class="fa fa-exclamation-circle"></i>
                     <span data-bind="text: $root.translations.provisional"></span>
-                </a>
+                </span>
                 <!--/ko-->
 
                 <!-- ko if: !!$parent.details -->
-                <a class="search-candidate-link" href="#" data-bind="click: showDetails">
+                <button class="search-candidate-link" data-bind="onEnterkeyClick, onSpaceClick, click: function(){showDetails();},
```

Comment:
I agree with you on the readability front, but had to do this in a few places because the methods would not fire on click. I suspect this is because I changed the elements from anchors to buttons